### ISSUE TITLE
batch normalization layer need to be corrected

### ISSUE THREAD

**Issue by safarisadegh** [immortal3/AutoEncoder-Based-Communication-System#3]
hi, i think you made a mistake in using keras BatchNormalization in autoencoder_7_4_symbol.ipynb. If you see in model summary you see that batch normalization layer has 14 trainable parameters that is because of scale and center parameters have True default value. I think you should use BatchNormalization as follows:
encoded2 = BatchNormalization(center=False, scale=False)(encoded1)

Thank you for your helpfull repo.
<img width="397" alt="summury" src="https://user-images.githubusercontent.com/37085305/52494586-a0fcbf80-2be3-11e9-8664-17d5e8aed018.PNG">

**Comment by immortal3** [comment]
Yes, It looks like I implemented it wrongly. But, You can refer to autoencoder_dynamic.ipynb for correct results. Closing this.
